src/resolvers/index.ts

```diff
@@ -26,4 +28,30 @@ export default {
   zapper,
   starknet,
   farcaster
-};
+} as const;
+
+export function resolve(
+  type: string,
```

Comment:
We could use `ResolverType` from utils.

We're always using a known type (a constrain from the express routes), maybe we could remove the fallback on line 40 below